docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -88,7 +93,19 @@ Complete the **G-Flags** section as follows:
 
   - [YB T-Server Configuration Flags](/preview/reference/configuration/yb-tserver/#configuration-flags)
 
-Accept default values for all of the remaining fields.
+Optionally, use the **Helm Overrides** section as follows:
+
+- Click **Add Kubernetes Overrides** to open the **Kubernetes Overrides** dialog shown in the following illustration:
+
+  ![img](/images/yb-platform/kubernetes-config66.png)
+
+- Using the sample format, specify the universe-level overrides for YB-Master and YB-TServer, as well as availability zone overrides.
```

Comment:
We should include the following details

1. Overrides are specified in yaml format, for example
```
master:
  podLabels:
    service-type: 'database'
```
2. Overrides need to be valid helm values from the [yugabyte helm chart](https://github.com/yugabyte/charts/blob/master/stable/yugabyte/values.yaml)
3. Availability zone level overrides only apply to pods that are deployed int that specific availability zone. To specify overrides for availability zone us-west-2a, one would specify yaml of the form

```
us-west-2a:
  master:
    podLabels:
       service-type: 'database'
```
4. When conflicting overrides are specified, the following order of precedence is used : universe availability zone level overrides, universe-level overrides, provider overrides